source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -25,17 +25,6 @@
      - ✓
      - ✓
 
-   * - 3.7
-     - ✓
-     - ✓
-     - ✓
-     - ✓
-     - ✓
-     - ✓
-     - ✓
-     - ✓
-     - ✓
-
    * - 3.6
```

Comment:
Question: It seems that only 3.6 doesn't start with **>=**, should it since all the other versions do?

Reply:
good question. I removed all of the >= entries since we have other versions ordered sequentially, it didn't make a lot of sense.